<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogoCreacionBaseDeDatos</class>
 <widget class="QDialog" name="DialogoCreacionBaseDeDatos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>177</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Creacion de bases de datos</string>
  </property>
  <widget class="QLabel" name="lbl_nombre_base_de_datos">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>161</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre de la Base de Datos:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_nombre_base_datos">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_crear_base_datos">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>341</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Crear base de datos</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_resultado">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>130</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
